src/components/table/SearchTable.tsx

```diff
@@ -2,71 +2,67 @@ import React from 'react';
-import { alpha, CustomTheme } from '@mui/material/styles';
-import { WithStyles, createStyles, withStyles } from '@mui/styles';
+import { alpha, Box, Link, styled, useTheme } from '@mui/material';
+import { CustomTheme } from '@mui/material/styles';
 
-const styles = (theme: CustomTheme) =>
-  createStyles({
-    search: {
-      height: 45,
-      width: '100%',
-      border: `1px solid ${theme.palette.common.border}`,
-      backgroundColor: theme.palette.common.selectedTextBackground,
-      '&:hover': {
-        backgroundColor: alpha(theme.palette.common.selection, 0.2),
-      },
-      position: 'relative',
-      borderRadius: '35px',
-      marginLeft: 0,
-      flex: '1 1 0',
-      display: 'flex',
-    },
-    searchIcon: {
-      color: theme.mixins.jadeLink.color,
-      paddingLeft: theme.spacing(1.8),
-      paddingTop: theme.spacing(1.3),
-    },
-    searchInput: {
-      paddingLeft: `${theme.spacing(3.7)} !important`,
-    },
-    inputRoot: {
-      width: '100%',
-    },
-    clearSearch: {
-      paddingRight: '15px',
-      height: '60%',
-      transform: 'translateY(30%)',
-      cursor: 'pointer',
-      ...theme.mixins.jadeLink,
-    },
-  });
+const SearchContainer = styled(Box)(({ theme }: { theme: CustomTheme }) => ({
+  height: '45px',
+  width: '100%',
+  border: `1px solid ${theme.palette.common.border}`,
+  backgroundColor: theme.palette.common.selectedTextBackground,
+  '&:hover': {
+    backgroundColor: alpha(theme.palette.common.selection, 0.2),
+  },
+  position: 'relative',
+  borderRadius: '35px',
+  marginLeft: 0,
+  flex: '1 1 0',
+  display: 'flex',
+}));
 
-interface IProps extends WithStyles<typeof styles> {
-  searchString: string;
-  onSearchStringChange: any;
-  clearSearchString: any;
+const ClearSearch = styled(Link)(({ theme }: { theme: CustomTheme }) => ({
+  paddingRight: '15px',
+  height: '60%',
+  transform: 'translateY(30%)',
+  cursor: 'pointer',
+  ...theme.mixins.jadeLink,
+}));
+
+interface IProps {
+  readonly searchString: string;
+  readonly onSearchStringChange: any;
+  readonly clearSearchString: any;
 }
 
-const SearchTable = withStyles(styles)(
-  ({ classes, searchString, onSearchStringChange, clearSearchString }: IProps) => (
-    <div className={classes.search}>
-      <div className={classes.searchIcon}>
+function SearchTable({ searchString, onSearchStringChange, clearSearchString }: IProps) {
+  const theme = useTheme() as CustomTheme;
+  return (
+    <SearchContainer theme={theme}>
+      <div
+        style={{
+          color: theme.mixins.jadeLink.color,
+          paddingLeft: theme.spacing(1.8),
+          paddingTop: theme.spacing(1.3),
+        }}
+      >
         <SearchSVG />
       </div>
       <InputBase
         placeholder="Search keyword or description"
-        classes={{
-          root: classes.inputRoot,
-          input: classes.searchInput,
+        sx={{
+          paddingLeft: '30px !important',
```

Comment:
The original used `${theme.spacing(3.7)}` instead of `30px`, did the original spacing no longer work here?